### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by porres**
document adjustments on trace, file and gatom help files

### CODE REVIEW

**Comment by porres** [comment]
@millerpuckette, please consider merging this ASAP. It has some crucial fixes like adding back sme help files that were deleted; it also includes trace and file in the help-intro.pd file, my review on the documentation of new objects, plus many other things, nothing too intrusive and just 'no brainer'.

I made a mistake and did not create a proper branch for the doc updates in my fork, so I need to get this merged before sending other PRs for other issues. 

thanks

**Review by Ant1r — COMMENTED** [review]
@porres I don't see any actual change here, and this doesn't match your commit messsage; didn't you miss something?

File: doc/5.reference/bng-help.pd [hunk 1]
```diff
@@ -524,19 +524,33 @@ objects., f 50;
...
+#000000 #000000;
+#X obj 557 364 bng 15 250 50 0 empty empty empty 17 7 0 10 #fcfcfc
+#000000 #000000;
+#X obj 553 208 s \$0-bang;
+#X obj 557 331 r \$0-bang;
+#X obj 561 250 bng 50 250 50 0 \$0-bang \$0-bang empty 17 7 0 10 #fcfcfc
+#000000 #000000;
+#X text 446 97 Feedback protection: iemguis can have the same name
+for both the send and receive symbols. This works without 'stack overflow'
+erros because they have feedbacck protection. See example below.,
```
[the remark this answers is not in the document]
Comment (porres) [reply doc/5.reference/bng-help.pd:540]: my damn keyboard is malfunctioning and doubling the 'c's sometimes :/ thanks

**Comment by porres** [comment]
> @porres I don't see any actual change here, and this doesn't match your commit messsage; didn't you miss something? (sorry this was related to [4f1b650](https://github.com/pure-data/pure-data/commit/4f1b650ad41eea80b5479b3bbaee3344f5858916) ; I'll delete this comment soon)

I see my changes there, maybe not on this commit, sorry for the mess

**Comment by Ant1r** [comment]
it's ok! 
there's still a typo in `list-help.pd`:
`list itrator` should be `list iterator`

**Comment by porres** [comment]
> it's ok! there's still a typo in `list-help.pd`: `list itrator` should be `list iterator`

fixed in 74c70ac

**Comment by porres** [comment]
I reverted my commit on changes I did to symbol-help.pd to solve conflicts now that develop branch got merged

@millerpuckette, please consider merging this to avoid more conflict complications, thanks

**Comment by porres** [comment]
It seems the commits got duplicated somehow when I reverted one of the commits :/

**Comment by porres** [comment]
Thanks! I can now create a proper documentation branch in my repository :)

**Pull request merged**